OpenMarket/OpenMarket/ViewController/MainViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+// MainViewController.swift
+// OpenMarket
+//
+//  Created by 써니쿠키, 메네 on 2022/11/22.
+//
+
+import UIKit
+
+final class MainViewController: UIViewController {
+    @IBOutlet private weak var listView: UIView!
+    @IBOutlet private weak var gridView: UIView!
+    @IBOutlet private weak var segmentControl: UISegmentedControl!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setUpSegmentControl()
+        gridView.isHidden = true
+    }
+    
+    private func setUpSegmentControl() {
+        segmentControl.backgroundColor = .systemBackground
+        segmentControl.setTitleTextAttributes([.foregroundColor: UIColor.white,
+                                               .backgroundColor: UIColor.systemBlue],
+                                              for: .selected)
+        segmentControl.setTitleTextAttributes([.foregroundColor: UIColor.systemBlue],
+                                              for: .normal)
+    }
+    
+    @IBAction private func changeView(_ sender: UISegmentedControl) {
+        if sender.selectedSegmentIndex == 0 {
+            listView.isHidden = false
+            gridView.isHidden = true
+            
+        } else {
+            listView.isHidden = true
+            gridView.isHidden = false
+        }
+    }
+}
```

Comment:
enum을 적절히 활용 해 볼 수 있지 않을까요?

```suggestion
enum SegmentedItemState {
case grid
case list
}
.
.
.
guard let selectedIndex = SegmentedItemState(rawValue: sender.selectedSegmentIndex) else { return }

switch selectedIndex {
// hidden 처리
}
```

Reply:
조언주신데로 SegmentedItemState 를 enum으로 정리했습니다 ㅎㅎ
enum으로 정리해놓으면 유지보수에 더 좋을 것 같고, switch문에서 list, grid 라고 쓸 수 있으니 명시적으로 느껴져서 너무 좋네요! 감사합니다 :)
 (https://github.com/yagom-academy/ios-open-market/pull/221/commits/1ce6f07e25143570d2b16aa7788b8ee1638b43f6)